source/cursorManager.py

```diff
@@ -179,7 +179,7 @@ def doFindText(self, text, reverse=False, caseSensitive=False, willSayAllResume=
 			if not willSayAllResume:
 				speech.speakTextInfo(info, reason=controlTypes.OutputReason.CARET)
 		else:
-			wx.CallAfter(gui.messageBox,_('text "%s" not found')%text,_("Find Error"),wx.OK|wx.ICON_ERROR)
+			wx.CallAfter(gui.messageBox,_('text "%s" not found')%text,_("Find Error"),wx.OK|wx.ICON_INFORMATION)

```

Comment:
Please lint your code changes

```suggestion
			wx.CallAfter(
				gui.messageBox,
				_('text "%s" not found') % text,
				_("Find Error"), wx.OK | wx.ICON_INFORMATION
			)